test/device_code/device_resume.test.js

```diff
@@ -392,7 +392,7 @@ describe('device interaction resume /device/:user_code/:uid/', () => {
         await this.agent.get(path)
           .expect(200)
           .expect('content-type', 'text/html; charset=utf-8')
-          .expect(/<body onload="javascript:document\.forms\[0]\.submit\(\)"/)
+          .expect(/<body>/)
```

Comment:
remove this line and add the one asserting that a self submitting code is present.